### ISSUE TITLE
SUPPORT: Nimrod twitching jaws 5/12/2019 on call

### ISSUE THREAD

**Issue by John-Holt-Tessella** [ISISComputingGroup/IBEX#5000]
The jaws on NIMROD still seem to be moving due to the noise on the encoder:

![image](https://user-images.githubusercontent.com/7570055/70070512-a2bcc700-15eb-11ea-9659-f13b5fbe5dfd.png)

Plan is to 
1.	Set the EDEL to 1mm which we will do tomorrow during your window of opportunity. 
2.	Extract from the db the motor positions and the jaws manager set points. From this we should be able to calculate where they should be and therefore see the errors over this cycle.

This should give us an idea of how large EDEL should be based on this cycle.

Do when T calls from NIMROD

**Comment by John-Holt-Tessella** [comment]
Run query

`%PYTHON% log_file_generator.py --start_time "2019-11-10T00:00:00" --delta_time 1 --point_count 2160000
 --host NDXNIMROD --filename_template log_nimrod_jaws.csv H-L1 IN:NIMROD:MOT:MTR0101.VAL H-R1 IN:NIMROD:MOT:MTR0102.VAL V-U1 IN:NIMROD:MOT:MTR0103.VAL V-L1 IN:NIMROD:MOT:MTR0104.VAL H-L2 IN:NIMROD:MOT:MTR0105.VAL H-R2 IN:NIMROD:MOT:MTR0106.VAL V-U2 IN:NIMROD:MOT:MTR0107.VAL V-L2 IN:NIMROD:MOT:MTR0108.VAL H-L3 IN:NIMROD:MOT:MTR0201.VAL H-R3 IN:NIMROD:MOT:MTR0202.VAL V-U3 IN:NIMROD:MOT:MTR0203.VAL V-L3 IN:NIMROD:MOT:MTR0204.VAL H-L4 IN:NIMROD:MOT:MTR0205.VAL H-R4 IN:NIMROD:MOT:MTR0206.VAL V-U4 IN:NIMROD:MOT:MTR0207.VAL V-L4 IN:NIMROD:MOT:MTR0208.VAL H-L5 IN:NIMROD:MOT:MTR0301.VAL H-R5 IN:NIMROD:MOT:MTR0302.VAL V-U5 IN:NIMROD:MOT:MTR0303.VAL V-L5 IN:NIMROD:MOT:MTR0304.VAL H-L6 IN:NIMROD:MOT:MTR0305.VAL H-R6 IN:NIMROD:MOT:MTR0306.VAL V-U6 IN:NIMROD:MOT:MTR0307.VAL V-L6 IN:NIMROD:MOT:MTR0308.VAL SAMPLE_HORIZONTAL IN:NIMROD:JAWMAN:SAMPLE:HGAP:SP SAMPLE_VERTICAL IN:NIMROD:JAWMAN:SAMPLE:VGAP:SP Moderator_Horizontal IN:NIMROD:JAWMAN:MOD:HGAP:SP Moderator_Vertical IN:NIMROD:JAWMAN:MOD:VGAP:SP`

in C:\Instrument\Apps\EPICS\ISIS\inst_servers\master\scripts

**Comment by John-Holt-Tessella** [comment]
T on NIMROD says no time today maybe tomorrow morning

**Comment by Tom-Willemsen** [comment]
EDELs changed to 1mm on all jaw axes NIMROD

Tris will call again tomorrow when we can run data collection script

**Comment by Tom-Willemsen** [comment]
Output of script is in `\\isis\shares\ISIS_Experimental_Controls\data for tickets\Ticket5000`

**Comment by DominicOram** [comment]
From Motion Control: can we check if this is still as bad when beam is off?

**Comment by John-Holt-Tessella** [comment]
Yes they can if they like :-)
